connect/src/com/telenor/connect/id/Claims.java

```diff
@@ -8,6 +8,7 @@
 
     public static final String OPENID = "openid";
     public static final String PROFILE = "profile";
```

Comment:
The documentation is unfortunately not accurate, since claims are separate from and more fine-grained than scope values. I will look into improving the docs. I'm sorry I forgot to do the review for UXUI-558 while being on parental leave.

We do only support the claims listed in https://gerrit.c.skunk-works.no/gitweb/?p=comoyoauth.git;a=blob;f=oauth-protocol/src/main/java/com/comoyo/oauth/protocol/Claim.java;hb=refs/heads/master, see also the documentation for the userinfo endpoint: http://docs.telenordigital.com/apis/connect/id/authentication.html#authorization-server-user-information.

Also, in the authorization server, we take some liberties with respect to how we interpret the claims request. More specifically, we ignore whether they are requested for userinfo or id_token and include them both places. We also map fine-grained claims to more course-grained scope values internally, so we are sometimes returning more claims than were requested.

It would be nice if you could update the claims functionality in the SDK to only include the supported claims, and possibly if you could support using both userinfo and id_token in the claims request? Consider doing this in a separate task/commit.